### ISSUE TITLE
Feature Request Import Meeting Participants

### ISSUE THREAD

**Issue by santiagoscutariloreal** [sawhney17/logseq-calendars-plugin#30]
Hello Team,

 It will be great if when import the meetings it also comes the participants of the meetings in order to use that to track and relate the meeting with other pages with the same people.

Thanks in advance

**Comment by sawhney17** [comment]
I think you can use the template for {{participants}}

**Comment by santiagoscutariloreal** [comment]
Hi @sawhney17 thanks for your answer. I tried {{participants}} but does not work (i don't know if this feature is not release yet or i am doing something wrong). 
Ideally, in a perfect world, it will be good to have both {{participants_mail}} or {{participants_name}} and also (because I try to have the history of what i talked with each contact) the ability to use [[]] to create page/link to each contact (I think could be a challenge for meeting with multiple contacts)

Thanks in advance

**Comment by sawhney17** [comment]
Yeah, I meant to say it doesn’t work. Was just curious how it hsouldlook like before I implement it.

**Comment by kennethaar** [comment]
A list with names and emails would be awesome.

**Comment by mbssai** [comment]
Yes would be amazing if possible add a list of names? I'm thinking like:

[[Name 1]] [[Name 2]] [[Name 3]]

@sawhney17 what do you think? :)
